sparta/example/CoreModel/gen_layouts.py

```diff
@@ -0,0 +1,87 @@
+import sys,os
+
+sys.path.append('../../../../helios/pipeViewer/scripts')
+from alf_gen.ALFLayout import ALFLayout
+
+if os.path.isfile("pipeout/location.dat") == False:
+    print('''ERROR: Cannot find pipeout/location.dat.  This tool cannot be run without a location file.
+    To generate one:
+        mkdir pipeout
+        ./sparta_core_example -i1 -z pipeout/
+    ''')
+    exit(255)
+
+try:
+    os.mkdir('layouts')
+except:
+    pass
+
+num_cycles = 55
```

Comment:
Variables intended to be constant should be named in all caps.

Reply:
Done